Forms/MainForm.cs

```diff
@@ -68,6 +73,14 @@ private bool OpenSelectISODialog()
       return true;
     }
 
+    private bool OpenExportCSVDialog()
```

Comment:
We should add a default name for the files we're exporting, for example when exporting the card properties table it could be card_properties.csv